### PR TITLE
Attach admins to stops

### PR DESCRIPTION
**Pull request by antoine-de**
attach the stop_areas to some admins

as a first step of stops2mimir we load all (for the moment) admins of mimir into a quadtree and we search for all the admins's boundary that contains the stop coordinate.

Note: For convenience purpose I added an intermediate type alias to mimir, we now have the index layout:
```
munin -> munin_addr -> munin_addr_dataset1 -> munin_addr_dataset1_20160101T123200
                   |-> munin_addr_dataset2 -> munin_addr_dataset2_20160101T123200
     |-> munin_admin -> munin_admin_dataset1 -> munin_admin_dataset1_20160101T123200
     |-> munin_street -> munin_street_dataset1 -> munin_street_dataset1_20160101T123200
```

### CODE REVIEW

File: tests/bragi_test.rs [hunk 1]
```diff
@@ -419,7 +449,25 @@ pub fn bragi_tests(es_wrapper: ::ElasticSearchWrapper) {
...
+    assert_eq!(get_value(stop, "city"), "Vaux-le-Pénil (77000)");
+    let admins = stop.get("administrative_regions").and_then(|a| a.as_array());
+    assert_eq!(admins.map(|a| a.len()).unwrap_or(0), 1);
+
+    // we query another stop, but this one is outside the range of an admin,
+    // we should get the stop, but with no admin attached to it
+    let response = get_results(bragi_get("/autocomplete?q=Far west station"));
+    assert_eq!(response.len(), 1);
+    let stop = response.first().unwrap();
+    info!("stop = {:?}", stop);
```
Comment (datanel) [inline tests/bragi_test.rs]: you can remove this line

File: README.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ To write a new test:
...
 Data are imported in multiple indexes with this structure:
 ```
-munin -> addr_dataset1 -> addr_dataset1_20160101T123200
-     |-> addr_dataset2 -> addr_dataset2_20160101T123200
-     |-> admin_dataset1 -> admin_dataset1_20160101T123200
-     |-> street_dataset1 -> street_dataset1_20160101T123200
+munin -> munin_addr -> munin_addr_dataset1 -> munin_addr_dataset1_20160101T123200
+                   |-> munin_addr_dataset2 -> munin_addr_dataset2_20160101T123200
+     |-> munin_admin -> munin_admin_dataset1 -> munin_admin_dataset1_20160101T123200
+     |-> munin_street -> munin_street_dataset1 -> munin_street_dataset1_20160101T123200
```
Comment (pbougue) [inline README.md]: would be nice to add stops to that example (and the one below)

File: tests/bragi_test.rs [hunk 3]
```diff
@@ -402,7 +425,14 @@ pub fn bragi_tests(es_wrapper: ::ElasticSearchWrapper) {
     let types = get_types(&geocodings);
     assert_eq!(count_types(&types, "poi"), 0);
 
+    // ******************************************
     // we then load a stop file
+    // the current dataset are thus:
+    // - sample-bano
+    // - bano-three_cities
+    // - osm_fixture.osm.pbf + pois
+    // stops.txt
```
Comment (pbougue) [inline tests/bragi_test.rs]: minor: missing `-`

File: tests/bragi_test.rs [hunk 4]
```diff
@@ -402,7 +425,14 @@ pub fn bragi_tests(es_wrapper: ::ElasticSearchWrapper) {
     let types = get_types(&geocodings);
     assert_eq!(count_types(&types, "poi"), 0);
 
+    // ******************************************
     // we then load a stop file
+    // the current dataset are thus:
+    // - sample-bano
+    // - bano-three_cities
+    // - osm_fixture.osm.pbf + pois
```
Comment (pbougue) [inline tests/bragi_test.rs]: `(including pois)` is clearer to me (same above)

File: README.md (hunk 2 above)
Comment (pbougue) [inline README.md]: Done

File: src/bin/bano2mimir.rs [hunk 5]
```diff
@@ -97,11 +100,18 @@ fn index_bano<I>(cnx_string: &str, dataset: &str, files: I)
...
+            info!("Administratives regions not found in es db for dataset {}. (error: {})",
+                  dataset,
+                  err);
+            vec![]
+        })
+        .into_iter()
+        .fold(AdminFromInsee::new(), |mut a_by_insee, a| {
+            a_by_insee.insert(a.id.to_string(), Rc::new(a));
+            a_by_insee
+        });
```
Comment (TeXitoi) [inline src/bin/bano2mimir.rs]:
```rust
    .map(|a| (a.id.to_string(), Rc::new(a))
    .collect();
```
instead of `fold`

**Pull request merged**